### ISSUE TITLE
False-positive UnusedArgument warning

### ISSUE THREAD

**Issue by printercu** [sds/slim-lint#61]
Hi!

I've found that block argument which is used as attribute value generates warning, like this:

```slim
- 2.times do |x|
  div attr=x
```

Can this be fixed?

**Comment by printercu** [comment]
Have also faced a warning with:

```slim
- if a.size == 2
  .first= a.first
  .last= a.last
- else
  = a.last

/ test.slim:3 [W] RuboCop: Move `a.last` out of the conditional.
/ test.slim:5 [W] RuboCop: Move `a.last` out of the conditional.
```

**Comment by printercu** [comment]
One more similar to first one:
```slim
= capture
  - style = :test
  div class=style

- capture
  - other_style = :test
  div class=other_style

/ gives warning only for the first part:
/ test.slim:2 [W] RuboCop: Useless assignment to variable - `style`.
```

**Comment by sds** [comment]
Thanks for the reports, @printercu.

I can't reproduce the issue for the first example you give:

```slim
- 2.times do |x|
  div attr=x
```

This doesn't output a warning for me.

I _can_ reproduce the issue for your other examples, however. 

Can I get you to confirm that you see an issue for this first case? If so, can you include the output of `slim-lint -V`?

**Comment by printercu** [comment]
I can't reproduce exactly the same case too :) I don't remember, maybe I've checked this only within template and it was wrapped into other block. So it's close to 3d case, but maybe slightly different:

```slim
= capture
  - 2.times do |x|
    div attr=x
```
